examples/oft-solana/README.md

```diff
@@ -10,14 +10,56 @@
 
 <h1 align="center">Omnichain Fungible Token (OFT) Solana Example</h1>
 
-## Setup
+## Requirements
+
+- Rust
+- Anchor v0.29
+- Solana CLI v.1.17.31
+- Docker
+- Node.js
 
-Ensure that you have Rust, Solana, and Anchor installed. If not, follow the instructions in the [official Solana guide](https://solana.com/docs/intro/installation#install-dependencies). On the page as well are instructions on creating a wallet and getting devnet SOL.
+## Setup
 
 We recommend using `pnpm` as a package manager (but you can of course use a package manager of your choice).
+
 [Docker](https://docs.docker.com/get-started/get-docker/) is required to build using anchor. We highly recommend that you use the most up-to-date Docker version to avoid any issues with anchor
 builds.
 
+:warning: You need anchor version `0.29` and solana version `1.17.31` specifically to compile the build artifacts. We are currently blocked by these issues from upgrading: [1](https://github.com/coral-xyz/anchor/issues/3089), [2](https://github.com/coral-xyz/anchor/issues/2835).
```

Comment:
Instead of saying "we're blocked", frame it as:

"Using higher Anchor and Solana versions can introduce unexpected issues during compilation."

After compiling the correct build artifacts, you can change the Solana version to higher versions.